src/test/java/com/google/sps/integration/ElectionInfoIT.java

```diff
@@ -25,9 +32,46 @@ public void testSetUp() {
   }
 
   /**
-   * TODO(anooshree): Resolve issue of being unable to test address input box locally due to API
-   * restrictions
+   * Tests if the submit button allows submission when the site is first loaded, but the user has
+   * not entered an address
    */
+  @Test
+  public void addressSubmission_onClick_blocksSubmissionWithoutAddress() {
+    driver.get("http://localhost:9876/electionInfo.html?state=ca&electionId=2000");
+    WebElement submitButton = driver.findElement(By.id("submit-address-button"));
+    submitButton.click();
+
+    Assert.assertEquals(
+        "http://localhost:9876/electionInfo.html?state=ca&electionId=2000", driver.getCurrentUrl());
+  }
+
+  @Test
+  public void addressSubmission_onClick_createAndUpdateQueryParams() throws InterruptedException {
```

Comment:
Sorry I think I misunderstood you earlier, I don't think it makes sense to submit this test if it can't pass maven verify.

I don't think the github system will let you merge these changes either?

Reply:
It passes maven verify, just not on github because config.js isn't pushed to github. I am pretty sure you can push regardless.

Reply:
You can just delete the test and leave a TODO explaining the problem. We'll punt on it for now, but it's bad to check something in that will permanently break the github presubmit tests.